### PR TITLE
Set receptor debug level to debug

### PR DESCRIPTION
**Pull request by kdelee**
This gives us more insight into what is happening in receptor

Julen from QE suggested we should do this

### CODE REVIEW

File: templates/install_vars.j2 [hunk 1]
```diff
@@ -3,6 +3,7 @@ admin_password: 'password'
 pg_password: 'password'
 pg_database: 'awx'
 pg_username: 'awx'
+receptor_log_level: {{ receptor_log_level|default("debug") }}
```
Comment (jhutar) [inline templates/install_vars.j2:6]: Is `debug` the default here?

Comment (kdelee) [reply templates/install_vars.j2:6]: yes

Comment (jainnikhil30) [reply templates/install_vars.j2:6]: isnt the default "info" ?
```
[root@perfscale-automation-controller-control1-vsi ~]# cat /etc/receptor/receptor.conf | grep -i log
# Log Level
- log-level: info
```
Or did you mean we should just default to debug no mater what ?

Comment (jhutar) [reply templates/install_vars.j2:6]: I do not know what is the factory-state default, but I just meant non-default value should come from config, not from config file, so if I do not care and not have it in the config, I get setup that is as much  "normal" as possible.

But still, this is a small thing :)

Comment (kdelee) [reply templates/install_vars.j2:6]: Oh yeah, the "factory" default is `info`. I want the ability to SET it to debug. I could do
`receptor_log_level: {{ receptor_log_level|default("info") }}`

Comment (kdelee) [inline templates/install_vars.j2:6]:
```suggestion
{% if receptor_log_level is defined %}
receptor_log_level:  {{ receptor_log_level }}
{% endif %}
```

**Review by jainnikhil30 — APPROVED** [review]

**Pull request merged**